### ISSUE TITLE
contributor.add script failling

### ISSUE THREAD

**Issue by mwickett** [kentcdodds/glamorous-website#172]
**Problem Description:** 
It's probably something I'm doing, but I can't seem to add myself as a contributor (because vanity)

Running `npm start contributors.add` fails after entering my gh username and selecting contribution types. The error is `Unexpected token } in JSON at position 2932`

I also tried adding myself in `.all-contributorsrc`, but the `contributors.generate` script fails during the commit husky process with 

```
TypeError: Cannot read property 'scriptName' of undefined
    at runPackageScript (/Users/michaelwickett/Sites/oss/glamorous-site/node_modules/nps/dist/index.js:95:35)
    at /Users/michaelwickett/Sites/oss/glamorous-site/node_modules/nps/dist/index.js:77:14
    at process._tickCallback (internal/process/next_tick.js:109:7) 
```

**Suggested Solution:** Hoping that someone else can confirm this is happening on their machine. Once I'm sure it's not just me, I'll happily dig deeper into what is going on here and hopefully fix it.

**Comment by tdeschryver** [comment]
Normally after #173 is merged, you can add yourself to the list!

**Comment by kentcdodds** [comment]
I've definitely seen others report this. Let's figure out what's up!

**Comment by paulmolluzzo** [comment]
@mwickett can you confirm this is this resolved by #173? Thanks!

**Comment by kentcdodds** [comment]
Oh yeah, I think it is. I'm going to go ahead and close this. Please re-open if it's not resolved.
